tools/releaseBuild/azureDevOps/WindowsBuild.yml

```diff
@@ -0,0 +1,513 @@
+jobs:
+
+- job: BuildJob
+  displayName: Build
+  condition: succeeded()
+  pool:
+    name: PowerShell
+    demands:
+      - Agent.Image
+      - USERNAME -equals issblder
+      - DotNetFramework
+  strategy:
+    matrix:
+      Build (x64,release):
+        BuildConfiguration: release
+        BuildPlatform: any cpu
+        Architecture: x64
+      Build (x86,release):
+        BuildConfiguration: release
+        BuildPlatform: any cpu
+        Architecture: x86
+      Build (arm,release):
+        BuildConfiguration: release
+        BuildPlatform: any cpu
+        Architecture: arm
+      Build (arm64,release):
+        BuildConfiguration: release
+        BuildPlatform: any cpu
+        Architecture: arm64
+      Build (fxdependent,release):
+        BuildConfiguration: release
+        BuildPlatform: any cpu
+        Architecture: fxdependent
+
+  steps:
+
+  - checkout: self
+    clean: true
+    persistCredentials: true
+
+  - task: PkgESSetupBuild@10
+    displayName: 'Initialize build'
+    env:
+      SYSTEM_ACCESSTOKEN: $(System.AccessToken)
+    inputs:
+      useDfs: false
+      productName: PowerShellCore
+      branchVersion: true
+      disableWorkspace: true
+      disableBuildTools: true
+      disableNugetPack: true
+    condition: and(succeeded(), eq(variables['Build.Reason'], 'Manual'))
```

Comment:
Instead of relying on build reason for deciding whether to sign or not, should we introduce a variable which defaults to false, but can be overridden?

Reply:
we can only sign if build reason is manual, but it might be good to centralize this logic

Reply:
At a later time I can do this across all builds. But this is Not a P0 at this point.